### ISSUE TITLE
Build error in crashreporter_posix.c

### ISSUE THREAD

**Issue by Teteros** [kmatheussen/radium#1246]
```
./compile_to_tmp.sh g++ -mfpmath=sse -msse2 audio/audio_instrument.cpp -c  `cat buildtype.opt` -Ibin/packages/gc-7.4.16/include -IQt/ -I/usr/include/python2.7 `cat flagopts.opt`  -Werr
or=array-bounds -msse2 -fomit-frame-pointer -DFOR_LINUX -DQT_X11EXTRAS_LIB -I/usr/include/qt/QtX11Extras -I/usr/include/qt -I/usr/include/qt/QtCore -DQT_GUI_LIB -I/usr/include/qt/QtGui
 -DQT_CORE_LIB  -DWITH_FAUST_DEV -DWITH_PD  -I/build/boost_1_70_0 -I/build/boost_1_67_0 -I/build/boost_1_63_0 -I /usr/include/vst36 -Wextra -Wno-unused-parameter -Wno-missing-field-ini
tializers -Wno-unknown-pragmas -fno-strict-aliasing -Wmissing-field-initializers -Wnull-dereference  -Wduplicated-cond -Wduplicated-branches -Wlogical-op -Wrestrict -DUSE_QT5 -Wmissing
-declarations -DNDEBUG   -Wno-class-memaccess  -Wno-parentheses -std=gnu++11 -DUSE_QT4 -DUSE_QIMAGE_BUFFER=1 `pkg-config --cflags Qt5Gui --cflags Qt5Network --cflags Qt5OpenGL --cflags
 Qt5Widgets --cflags Qt5WebKitWidgets --cflags Qt5WebKit --cflags Qt5Concurrent` -Ibin/packages/qhttpserver-master/src -I/build/radium/src/radium-5.9.98/bin/packages/QScintilla_gpl-2.1
0.8/Qt4Qt5 -DQHTTPSERVER_EXPORT  -fPIC -Wno-deprecated-copy
          common/stacktoucher.c  compiled. (0s)
"`/build/radium/src/radium-5.9.98/find_moc_and_uic_paths.sh moc`" audio/MultiCore.cpp >audio/mMultiCore.cpp
/usr/bin/moc
/usr/bin/uic
In file included from crashreporter/crashreporter_posix.c:33:
crashreporter/backtrace-symbols.c: In function ‘find_address_in_section’:
crashreporter/backtrace-symbols.c:136:7: warning: implicit declaration of function ‘bfd_get_section_flags’; did you mean ‘bfd_set_section_flags’? [-Wimplicit-function-declaration]
  136 |  if ((bfd_get_section_flags(abfd, section) & SEC_ALLOC) == 0)
      |       ^~~~~~~~~~~~~~~~~~~~~
      |       bfd_set_section_flags
crashreporter/backtrace-symbols.c:139:8: warning: implicit declaration of function ‘bfd_get_section_vma’; did you mean ‘bfd_set_section_vma’? [-Wimplicit-function-declaration]
  139 |  vma = bfd_get_section_vma(abfd, section);
      |        ^~~~~~~~~~~~~~~~~~~
      |        bfd_set_section_vma
crashreporter/backtrace-symbols.c:143:26: warning: passing argument 1 of ‘bfd_section_size’ from incompatible pointer type [-Wincompatible-pointer-types]
  143 |  size = bfd_section_size(abfd, section);
      |                          ^~~~
      |                          |
      |                          bfd * {aka struct bfd *}
In file included from crashreporter/backtrace-symbols.c:56,
                 from crashreporter/crashreporter_posix.c:33:
/usr/include/bfd.h:1205:35: note: expected ‘const asection *’ {aka ‘const struct bfd_section *’} but argument is of type ‘bfd *’ {aka ‘struct bfd *’}
 1205 | bfd_section_size (const asection *sec)
      |                   ~~~~~~~~~~~~~~~~^~~
In file included from crashreporter/crashreporter_posix.c:33:
crashreporter/backtrace-symbols.c:143:9: error: too many arguments to function ‘bfd_section_size’
  143 |  size = bfd_section_size(abfd, section);
      |         ^~~~~~~~~~~~~~~~
In file included from crashreporter/backtrace-symbols.c:56,
                 from crashreporter/crashreporter_posix.c:33:
/usr/include/bfd.h:1205:1: note: declared here
 1205 | bfd_section_size (const asection *sec)
      | ^~~~~~~~~~~~~~~~
```
Radium (release build) 5.9.98 + 94af142
Same err on both gcc 9.2.1 and clang 9.0.1
Arch Linux x64

**Comment by kmatheussen** [comment]
Seems to be a change the API of bfd if gdb 9.x is installed.

**Comment by kmatheussen** [comment]
Should be fixed in 37df0d0a2c5263c4be22c56ba6ddcbd65ef16371

**Comment by Teteros** [comment]
Works, thanks!
